exporter/datadogexporter/traces_exporter_test.go

```diff
@@ -290,6 +290,8 @@ func TestNewTracesExporter(t *testing.T) {
 }
 
 func TestPushTraceData(t *testing.T) {
+	t.Skip("Skipping as a result of https://github.com/open-telemetry/opentelemetry-collector-contrib/issues/17037")
```

Comment:
I've marked this as Skip for now @mx-psi , and if you could revert once your change has gone through?

Reply:
Sounds good to me, I opened #17045 to track fixing this and assigned to me